warehouse/email/__init__.py

```diff
@@ -342,6 +342,16 @@ def send_basic_auth_with_two_factor_email(request, user, *, project_name):
     return {"project_name": project_name}
 
 
+@_email(
+    "two-factor-not-yet-enabled",
+    allow_unverified=True,
+    # may repeat at 15 days due to table cleanup.
+    repeat_window=datetime.timedelta(days=90),
```

Comment:
should we explicitly set a repeat_window of 14 instead of implicitly accepting that this may be outside of our email table cleanup? this comment could become a lie if we change cleanup interval.

Reply:
Yeah, I like that better. It also helps if there's a transient issue with their email service (bounce) that's resolved within the 90 day window, AND they perform another release 14 days later, they'll get another email scheduled.